### PR TITLE
DDF-4154 Only allow number input for Intrigue point-radius latlon

### PR DESCRIPTION
**Pull request by ethantmanns**
#### What does this PR do?
Prevents users from entering other text into the lat/lon and radius fields that can cause issues deserializing a saved search.

#### Who is reviewing it? 

@Variadicism 
@SmithJosh 
@tyler30clemens 

#### Select relevant component teams: 
https://github.com/orgs/codice/teams
#### Ask 2 committers to review/merge the PR and tag them here.

@andrewkfiedler
@djblue
@figliold
@tbatie

#### How should this be tested?

* Build UI
* Attempt to enter non-numbers into basic search point-radius lat/lon
* Verify you can't

#### Any background context you want to provide?
Entering non-numbers resulted in them being saved in saved searches and caused errors to be thrown during deserialization.

#### What are the relevant tickets?
[DDF-4154](https://codice.atlassian.net/browse/DDF-4154)

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: ui/packages/catalog-ui-search/src/main/webapp/react-component/location/point-radius.js [hunk 1]
```diff
@@ -9,10 +9,10 @@ const PointRadiusLatLon = (props) => {
     const { lat, lon, radius, radiusUnits, cursor } = props;
     return (
         <div>
-            <TextField label="Latitude" value={lat} onChange={cursor('lat')} addon="°" />
-            <TextField label="Longitude" value={lon} onChange={cursor('lon')} addon="°" />
+            <TextField type="number" label="Latitude" value={lat} onChange={cursor('lat')} addon="°" />
+            <TextField type="number" label="Longitude" value={lon} onChange={cursor('lon')} addon="°" />
             <Units value={radiusUnits} onChange={cursor('radiusUnits')}>
-                <TextField label="Radius" value={radius} onChange={cursor('radius')} />
+                <TextField type="number" label="Radius" value={radius} onChange={cursor('radius')} />
```
Comment (willwill96) [inline ui/packages/catalog-ui-search/src/main/webapp/react-component/location/point-radius.js]: Should we add `min="0"` to this?

Comment (ethantmanns) [reply ui/packages/catalog-ui-search/src/main/webapp/react-component/location/point-radius.js]: I'm not opposed to that

**Comment by tyler30clemens** [comment]
Hero ✔️

**Comment by shaundmorris** [comment]
SonarQube did not report status, but wouldn't have found anything on this PR anyway.

**Pull request merged**